app-shells/fish/fish-4.0.0.ebuild

```diff
@@ -104,7 +104,7 @@ else
 		https://github.com/fish-shell/fish-shell/releases/download/${MY_PV}/${MY_P}.tar.xz
 		${CARGO_CRATE_URIS}
```

Comment:
If you want, you can use this Gentoo infra for tarball generation of crates automatically

https://github.com/gentoo-crate-dist/fish-shell/releases/download/4.0.0/fish-shell-4.0.0-crates.tar.xz

```shell
https://github.com/gentoo-crate-dist/fish-shell/releases/download/${PV}/fish-shell-${PV}-crates.tar.xz
```